internals/overlord/checkstate/manager_test.go

```diff
@@ -185,7 +184,6 @@ func (s *ManagerSuite) TestCheckCanceled(c *C) {
 	stopChecks(c, mgr)
 
 	// Ensure command was terminated (output file didn't grow in size)
-	time.Sleep(50 * time.Millisecond)
```

Comment:
stopChecks() calls PlanChanged(), which terminates the previous checks synchronously now, so this is not needed.